src/components/pages/CompanyData/components/CompanyAddressList.tsx

```diff
@@ -144,11 +154,12 @@ export const CompanyAddressList = ({
   const getStatusColor = (status: string | undefined) => {
     if (status === SharingStateStatusType.Success) {
       return 'success'
-    } else if (
-      status === SharingStateStatusType.Pending ||
-      status === SharingStateStatusType.Initial
-    ) {
+    } else if (status === SharingStateStatusType.Initial) {
       return 'warning'
+    } else if (status === SharingStateStatusType.Pending) {
```

Comment:
it has been in the code before but we don't need `else` in cases where the previous code will return 100% sure.

instead of
```
if (a) {
  return x
} else if (b) {
  return y
} else {
  return z
}
```
simply write
```
if (a)
  return x
if (b)
  return y
return z
```